tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "strict": true,
     "target": "es2020"
   },
-  "include": ["src", "tests"]
+  "include": ["src", "tests"],
```

Comment:
fyi, I think the last comma might have to go (maybe I messed that up) since json technically doesn't have extra commas